WalletWasabi.Backend/Controllers/BlockchainController.cs

```diff
@@ -48,6 +48,29 @@ public BlockchainController(IMemoryCache memoryCache, Global global)
 
 	public Global Global { get; }
 
+	/// <summary>
+	/// Get all fees.
+	/// </summary>
+	/// <param name="estimateSmartFeeMode">Bitcoin Core's estimatesmartfee mode: ECONOMICAL/CONSERVATIVE.</param>
+	/// <returns>A dictionary of fee targets and estimations.</returns>
+	/// <response code="200">A dictionary of fee targets and estimations.</response>
+	/// <response code="400">Invalid estimation mode is provided, possible values: ECONOMICAL/CONSERVATIVE.</response>
+	[HttpGet("all-fees")]
+	[ProducesResponseType(200)]
+	[ProducesResponseType(400)]
+	[ResponseCache(Duration = 300, Location = ResponseCacheLocation.Client)]
```

Comment:
This is wrong. Remove it.

Reply:
Done. I hope you only referred to the `ResponseCache` attribute.